pkg/controllers/controllers.go

```diff
@@ -106,7 +108,8 @@ func Register(ctx context.Context, systemNamespace, controllerName string, cfg c
 		klog.Infof("Starting helm controller in namespace %s", systemNamespace)
```

Comment:
And same for the global (non-namespaced) log message up above.
```suggestion
		klog.Infof("Starting %s in namespace %s", controllerName, systemNamespace)
```